rootfs/etc/nginx/lua/test/balancer/sticky_test.lua

```diff
@@ -55,17 +55,7 @@ describe("Sticky", function()
       it("returns an instance containing the corresponding cookie name", function()
         local sticky_balancer_instance = sticky:new(test_backend)
         local test_backend_cookie_name = test_backend.sessionAffinityConfig.cookieSessionAffinity.name
-        assert.equal(sticky_balancer_instance.cookie_name, test_backend_cookie_name)
-      end)
-    end)
-
-    context("when backend does not specify cookie name", function()
```

Comment:
Why is this test not necessary anymore?

Reply:
I removed it because the actual cookie name wasn't being stored on the balancer anymore. I've added the test back and made it use a method on the balancer to check the cookie name.